package.json

```diff
@@ -2,24 +2,18 @@
   "name": "@universityofwarwick/statuspage-widget",
   "version": "0.0.8",
   "dependencies": {
-    "@fortawesome/fontawesome-svg-core": "1.2.34",
-    "@fortawesome/pro-light-svg-icons": "5.15.2",
-    "@fortawesome/react-fontawesome": "0.1.14",
     "@testing-library/jest-dom": "5.11.9",
     "@testing-library/react": "11.2.5",
     "@testing-library/user-event": "12.7.0",
-    "abortcontroller-polyfill": "1.7.1",
     "node-sass": "5.0.0",
```

Comment:
Can all the build and test deps go into devDependencies, particularly node-sass which a few of our app builds end up compiling from source which takes forever then never uses it